ecl/eclcc/eclcc.hpp

```diff
@@ -96,6 +96,7 @@ const char * const helpText[] = {
     "!   -pch          Generate precompiled header for eclinclude4.hpp",
 #endif
     "!   -P <path>     Specify the path of the output files (only with -b option)",
+    "!   --releasemem  Fully clean up memory (useful when checking for memory leaks)",
```

Comment:
'useful'? I'm not clear on how valgrind (or similar) catches mem leaks, but under what condition is it at all possible to do so without using this option? E.g. should 'useful' -> 'necessary'?